Tests/UnitTests/Paywalls/Components/Properties/CornerRadiusesPropertyTests.swift

```diff
@@ -0,0 +1,54 @@
+import Nimble
+@testable import RevenueCat
+import XCTest
+
+#if PAYWALL_COMPONENTS
+
+class CornerRadiusesPropertyTests: TestCase {
+
+    func testAllValues() throws {
+        let json = """
+        {
+            "top_leading": 5,
+            "top_trailing": 5,
+            "bottom_leading": 5,
+            "bottom_trailing": 5
+        }
+        """
+
+        _ = try JSONDecoder.default.decode(
```

Comment:
For asserting that an optional is not `nil` in a test, there's the [XCTUnwrap](https://developer.apple.com/documentation/xctest/3380195-xctunwrap).
```
XCTUnwrap(try JSONDecoder.default.decode(...))
```